src/components/Calculator.css

```diff
@@ -0,0 +1,26 @@
+.bgOrange {
+  background-color: orange;
+}
+
+.bgLGrey {
+  background-color: lightgrey;
+}
+
+.grid {
+  background-color: aliceblue;
+  text-align: center;
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  gap: 1px;
+}
```

Comment:
- [ ] Your design does not match the given design, kindly fix this. The given design is centered to the middle form the left and the right.

GIven design
![image](https://user-images.githubusercontent.com/81932478/198567164-7c3644a0-88f3-4ae1-8f47-c3f176c56fae.png)

Your design
![image](https://user-images.githubusercontent.com/81932478/198567322-f75df57f-599f-4938-87aa-6929e305f7ff.png)

- [ ] I went ahead and tried playing around with you CSS and when I add the following properties your design looks better. And then youre able to see some issues with your design that were hidden due to the size like the bottom buttons. Kindly fix this you're free to use other solutions

![image](https://user-images.githubusercontent.com/81932478/198568494-027eb7a7-265e-47b3-92c7-91c9ebb6b87a.png)

#### Note in the screenshot above I add a `width` and `height` property and your design looks as follows:
![image](https://user-images.githubusercontent.com/81932478/198569473-dcf9336a-58d7-46fa-bb34-0fe1be1e3177.png)